### ISSUE TITLE
Fork not needed anymore

### ISSUE THREAD

**Issue by fsafq41** [Vargol/ffmpeg-apple-arm64-build#32]
I assume that this fork is not needed anymore as the official version from martinr92 you linked as your source has now Apple Silicon specific builds including the NEON patch for x265, correct?

**Comment by Vargol** [comment]
Yep, all the NEON code has been integrated into the source and builds by default now so any source of ffmpeg should one as fast as my builds.  
I basically keep it around for the Topaz Labs integration as the version they use can't have any of the GPL'd codecs in.
